src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -211,6 +213,45 @@ export const BulkEditListFilters = ({
     }
   }, [location.search]);
 
+
+  const getisDropzoneDisabled = (capabilitie) => {
```

Comment:
Maybe we need to make unified function that can take capabilitie and default perm, but i'm not sure